docs/source/usersguide/geometry.rst

```diff
@@ -0,0 +1,353 @@
+.. _usersguide_geometry:
+
+=================
+Defining Geometry
+=================
+
+.. currentmodule:: openmc
+
+--------------------
+Surfaces and Regions
+--------------------
+
+The geometry of a model in OpenMC is defined using `constructive solid
+geometry`_ (CSG), also sometimes referred to as combinatorial geometry. CSG
+allows a user to create complex regions using Boolean operators (intersection,
+union, and complement) on simpler regions. In order to define a region that we
+can assign to a cell, we must first define surfaces which bound the region. A
+surface is a locus of zeros of a function of Cartesian coordinates
+:math:`x,y,z`, e.g.
+
+- A plane perpendicular to the :math:`x` axis: :math:`x − x_0 = 0`
+- A cylinder perpendicular to the :math:`z` axis: :math:`(x − x_0)^2 + (y −
+  y_0)^2 − R^2 = 0`
+- A sphere: :math:`(x − x_0)^2 + (y − y_0)^2 + (z − z_0)^2 − R^2 = 0`
+
```

Comment:
Is it possible to add some whitespace between these bullets and the paragraph below?